pages/linux/rmlint.md

```diff
@@ -0,0 +1,20 @@
+# rmlint
+
+> Extremely fast tool to remove remove duplicates and other lint from your filesystem.
+> More information: <https://rmlint.readthedocs.io/en/latest/index.html>.
+
+- Check directory for duplicates, empty files, and other lint:
+
+`rmlint {{path/to/directory}}`
+
+- Delete lint files found by an rmlint run:
```

Comment:
```suggestion
- Delete duplicate files found by an execution of `rmlint`:
```

Reply:
Applied in [`9ad0fcb`](https://github.com/tldr-pages/tldr/pull/9090/commits/9ad0fcb8a887567c297e00bde72677c0c5b5b768).